pkg/armhelpers/utils/util.go

```diff
@@ -160,3 +160,26 @@ func GetK8sVMName(p *api.Properties, agentPoolProfile *api.AgentPoolProfile, age
 
 	return "", errors.Errorf("Failed to reconstruct VM Name")
 }
+
+// IsVMSSUpgradeable - checks if the current vm(in case of VMA) or vm instance (in case of VMSS)
+// is specified in the upgradeMap.
+func IsVMSSUpgradeable(upgradeMap map[string]bool, vmssName string) bool {
```

Comment:
I don't think this belongs in the `utils` package.

Reply:
what would be a good place for it ?

Reply:
can we have it in the `kubernetesupgrade` package?